### ISSUE TITLE
Ethos suite upgrade to 1.5.2 fails on mac

### ISSUE THREAD

**Issue by GilMSheinfeld** [FrSkyRC/ETHOS-Feedback-Community#3677]
Hi,
Upgraded ethos suite from 1.5.1 to 1.5.2 on mac, using the suite upgrade. The upgrade failed. There was a JavaScript error after install, and when connecting the X20S to upgrade from 1.5.1 to 1.5.2 the upgrade failed.
I manually copied the ethos suite 1.5.2 dmg over the existed suite, and that solved the issue.

**Comment by Nicholas-Luoyi** [comment]
Would you please share the screenshot of the javascript error here if it is reproducible for you?

**Comment by spoke2570** [comment]
I just wiped all ETHOS files from my Mac, manually installed Suite 1.5.1, and used the built in update feature to get to Suite 1.5.2. Suite crashed when this task was done, When I reopened it 1.5.2 was running. Below is the report from my OS and a screen shot. 

[OS 12.7.4.txt](https://github.com/FrSkyRC/ETHOS-Feedback-Community/files/14731606/OS.12.7.4.txt)

![Screen Shot 2024-03-23 at 8 36 23 AM](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/9330938/e3762cbd-0dc7-4f29-aaf2-5a1c1935806d)
